Rakefile

```diff
@@ -29,7 +29,7 @@ namespace :docs do
   desc "Clean/remove the generated YARD Documentation cache"
   task :clean do
     original_dir = Dir.pwd
-    Dir.chdir(File.expand_path(File.dirname(__FILE__)))
+    Dir.chdir(File.expand_path(__dir__))
```

Comment:
we use `__FILE__` in places like this because `rake` can be run from a different dir from where the Rakefile is.  e.g. if you run in a child dir, it looks for the rakefile in `./` then in `../`.  also `rake -f`